### DISCUSSION TITLE
How to show a specific floating window on every workspace?

### DISCUSSION THREAD

**Discussion by moshpirit** [i3/i3#5934] [Q&A] [answered]
Firefox allows us through Picture-in-Picture to show videos on a floating window on top of everything to watch them while doing something else. I like it to be shown on every workspace, so if I change from ws to ws, I don't have to be moving it constantly. How can I set up a binding key to toggle telling i3 that I want this window to be shown everywhere or act as a regular window?

**Comment by okraits** [comment, marked as answer]
Use a sticky window: https://i3wm.org/docs/userguide.html#_sticky_floating_windows

**Reply by moshpirit** [reply]
Thanks! It worked greatly!

Just in case anyone else wants to see the solution for this specific case (or myself in the future):

```
bindsym $mod+y sticky toggle

# make a terminal sticky (shows in all workspaces)
for_window [instance=Picture-in-Picture] sticky enable
```
